src/js/AuditRule.js

```diff
@@ -47,13 +48,26 @@ axs.AuditRule = function(spec) {
     /**
      * Is this element relevant to this audit?
      * @param {Element} element A potential audit candidate.
+     * @param {Object} flags
```

Comment:
As I read it, `flags` is actually optional. If this is the case, let's annotate it as such (`{Object=}`) and name accordingly to be consistent with other optional params.